ReactiveCocoa/NSObject+RACDescription.m

```diff
@@ -48,3 +48,15 @@ - (NSString *)rac_description {
 }
 
 @end
+
+NSString *rac_description(id object) {
+	if ([object respondsToSelector:@selector(rac_description)]) {
+		return [object rac_description];
+	} else {
+		if (getenv("RAC_DEBUG_SIGNAL_NAMES") != NULL && [object respondsToSelector:@selector(description)]) {
```

Comment:
Are you sure about this? As far as I can see, ``-description`` is a property implemented on ``NSObject`` (and also by some other base classes) but there is no guarantee that it would be implemented for an ``id`` object.